src/Renderer/MapScaleRenderer.cpp

```diff
@@ -28,6 +28,7 @@
 #include "Look/OverlayLook.hpp"
 #include "Util/StaticString.hxx"
 #include "Formatter/UserUnits.hpp"
+#include "Pan.hpp"
```

Comment:
This is a layering violation. The MapScaleRenderer is generic code and shouldn't depend on this library.

Reply:
I changed it so that now it offsets the reference PixelRect from GlueMapwindow's DrawMapScale. The MapScaleRenderer is now the same is it was before.